src/api/globals/fixture.js

```diff
@@ -37,6 +38,41 @@ export default class Fixture {
         return this;
     }
 
+    httpAuth (username, password, domain, workstation) {
```

Comment:
Why you decided to use arguments instead of descriptive object: http://www.codereadability.com/what-are-javascript-options-objects/? Let's stick with options object instead